src/gtk/render-asset.sh

```diff
@@ -5,12 +5,12 @@ RENDER_SVG="$(command -v rendersvg)" || true
 INKSCAPE="$(command -v inkscape)" || true
 OPTIPNG="$(command -v optipng)" || true
 
-if [ -n "$INKSCAPE" ] ; then
```

Comment:
this shouldn't be removed

that will break the script if inkscape is not installed

Reply:
@actionless I removed it as it wasn't in the other two similarly coded render scripts. Seems ok (and less complex IMO) because it only sets another option and isn't used to check whether to actually run inkscape.

Reply:
Ah, we have `set -ueo pipefail` in the scripts, so the script will be stopped if inkscape is not installed (but rendersvg is installed), right? I forgot about it. Thanks for pointing it out, @actionless. I will file a PR based on your review.